packages/rxjs/spec/operators/share-spec.ts

```diff
@@ -856,89 +855,86 @@ describe('share', () => {
           expectObservable(result, subscription).toBe(expected);
           expectSubscriptions(source.subscriptions).toBe(sourceSubs);
         });
-
-        expect(onUnhandledError).to.have.been.calledOnce;
```

Comment:
These tests were errant, in that they were able to check to see if `onUnhandledError` was dispatched synchronously, which it was never supposed to be.